pom.xml

```diff
@@ -215,10 +215,19 @@
         <groupId>com.diffplug.spotless</groupId>
         <artifactId>spotless-maven-plugin</artifactId>
         <version>2.12.2</version>
+        <executions>
+          <execution>
+            <phase>validate</phase>
+            <goals>
+              <goal>check</goal>
+            </goals>
+          </execution>
+        </executions>
         <configuration>
           <java>
             <googleJavaFormat>
-              <version>1.11.0</version>
+              <!-- Latest version which runs on java8 -->
+              <version>1.7</version>
```

Comment:
why a lower version?

Reply:
The latest versions won't run on java8, which is what CI is running with.  Something about a newer version of the java class format.